tests/cache/tests.py

```diff
@@ -1011,7 +1010,6 @@ def get_response(req):
             return response
 
         update_middleware = UpdateCacheMiddleware(get_response)
-        update_middleware.cache = cache
```

Comment:
`FetchFromCacheMiddleware` and `UpdateCacheMiddleware` don't use `default` cache by default. We should override `CACHE_MIDDLEWARE_ALIAS` to make sure they will use the same cache as before. We can do this in a separate commit. For example:
```diff
diff --git a/tests/cache/tests.py b/tests/cache/tests.py
index 29e527d4e6..13ff0a45aa 100644
--- a/tests/cache/tests.py
+++ b/tests/cache/tests.py
@@ -994,9 +994,9 @@ class BaseCacheTests:
         self.assertEqual(caches['custom_key'].get('answer2'), 42)
         self.assertEqual(caches['custom_key2'].get('answer2'), 42)
 
+    @override_settings(CACHE_MIDDLEWARE_ALIAS=DEFAULT_CACHE_ALIAS)
     def test_cache_write_unpicklable_object(self):
         fetch_middleware = FetchFromCacheMiddleware(empty_response)
-        fetch_middleware.cache = cache
 
         request = self.factory.get('/cache/test')
         request._cache_update_cache = True
@@ -1011,7 +1011,6 @@ class BaseCacheTests:
             return response
 
         update_middleware = UpdateCacheMiddleware(get_response)
-        update_middleware.cache = cache
         response = update_middleware(request)
 
         get_cache_data = fetch_middleware.process_request(request)

```

Reply:
I used your suggestion. I could change `middlewaare.cache_alias` instead of `middleware.cache` to avoid the property issue, but I preferred your solution.